Hometask4.playground/Contents.swift

```diff
@@ -0,0 +1,84 @@
+//Создать словарь, где ключ - название города/страны, значение - интересный факт и континент, где распорожен город/страна (подумайте, какой тип данных лучше использовать для значения).
+
+import Foundation
+
+var worldPlaceDictionary = ["Russia": ("The biggest country in the world", "Eurasia"),
+                            "Portugal": ("One of the most safety country in the world","Europe"),
+                            "Australia": ("The same as Europe but without migrants from near East","Australia"),
+                            "Netherlands": ("Gaaga", "Europe"),
+                            "Belarus": ("Misnk is one of the most clean city", "Europe"),
+                            "Taiwan": ("The main center of microconductor manufactor", "Asia"),
+                            "Japan": ("Neutral country", "Asia"),
+                            "USA": ("Top 10 companies with the biggest capitalisation placed in USA", "North America")]
+
+//1. Добавить новый элемент в словарь, поменять два существующих элемента в словаре, удалить один элемент из словаря.
+
+worldPlaceDictionary["Minsk"] = ("1067 battle on Nemiga", "Europe") // add a new element
+
+worldPlaceDictionary["Russia"]    = ("Putinka", "Eurasia")
+worldPlaceDictionary["Australia"] = ("A1", "Australia") // change elements in dictionary
+
+worldPlaceDictionary.removeValue(forKey: "Minsk")
+worldPlaceDictionary["Netherlands"] = nil // remove value from the Dictionary
+
+//2. Создать отдельный словарь с названием городов/стран с длиной более 6 букв из первоначального словаря.
+
+var dictCitiesLongName = [String: (String, String)] ()
+
+dictCitiesLongName = worldPlaceDictionary.filter({ $0.key.count > 6 })
+
+//3.  Создать массив интересных фактов в формате (“название города/страны: факт”) из первоначального словаря, отсортировав их по длине строки по убыванию
+
+var countriesFactsArray = [String]()
+
+for (country, fact) in worldPlaceDictionary {
+    countriesFactsArray.append("\(country): \(fact.0)")
+    countriesFactsArray = countriesFactsArray.sorted(by: { $0.count > $1.count })
```

Comment:
It's better to move this line out of the cycle. This way it's performed in every iteration. One call after all elements are added is enough

Reply:
Line moved out of cycle